modules/lobbies/scripts/join.ts

```diff
@@ -25,6 +28,15 @@ export async function run(
 	ctx: ScriptContext,
 	req: Request,
 ): Promise<Response> {
+	await ctx.modules.captcha.guard({
+		key: "lobbies.join",
+		period: 15,
```

Comment:
period & requests should be configurable in config.ts and have defaults in the default config